src/NexusMods.Abstractions.Loadouts.Synchronizers/ALoadoutSynchronizer.cs

```diff
@@ -1148,10 +1146,17 @@ private bool ActionIngestFromDisk(Dictionary<GamePath, SyncNode> syncTree, Loado
             }
         }
 
+        // Update locator IDs before building the sync tree
+        loadout = await UpdateLocatorIds(loadout);
```

Comment:
I think `UpdateLocatorIds(loadout)` might need to go before the `await ActivateLoadout(loadout);`  (L1144) during the loadout switching code.

Otherwise if there was a game update, and the user tried to switch to a different loadout we would:
- Correctly deactivate the previous Loadout, updating its `LocatorIds`
- Reset the game to the new `LocatorIds` vanilla state
- Try to Activate the other loadout against a vanilla game state that doesn't match the loadout locatorIds (still on the old game version).
- This breaks the Activate code which assumes we are doing a clean Apply of a loadout against its matching vanilla state.